5_kata.py

```diff
@@ -0,0 +1,14 @@
+#https://www.codewars.com/kata/58f6000bc0ec6451960000fd/train/python
+
+def sel_reverse(arr,l):
+    if l>0 and len(arr) > l:
+        qty = len(arr)
+        new_list = [(arr[i:i+l])[::-1] for i in range(0,qty,l)]
```

Comment:
Here also please add the spaces between mathematical operators AND comas :) 
This line should look like this: `new_list = [(arr[i:i + l])[::-1] for i in range(0, qty, l)]` after reformatting it.